WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -242,6 +246,18 @@ private void resetFragments() {
         }
     }
 
+    private void moderateCommentOnActivityResult(Intent data) {
+        try {
+            Note note = SimperiumUtils.getNotesBucket().get(StringUtils.notNullStr(data.getStringExtra
```

Comment:
Looking through other places that call `getNotesBucket()`, it's possible for it to return null. So this needs a null check.